### DISCUSSION TITLE
Welcome to UltimateTrainingModpack Discussions!

### DISCUSSION THREAD

**Discussion by jugeeya** [jugeeya/UltimateTrainingModpack#189] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dgrbrady** [comment]
Is there any plan to improve the info the saved states can capture? As a Hero main, I would LOVE if I could save a state with accelerattle on and when it runs out I know I can just reset and keep it active. Having an option in the GUI to toggle Hero buffs would also work, but I feel like it might be easier to implement in the save states? I would even work on it myself but for some reason I can't get it to build locally :/ I'm a Typescript/web dev though so I'm a little out of my element here haha

**Reply by jugeeya** [reply]
This is done in latest! Sorry I never saw this!
